Program.cs

```diff
@@ -603,6 +617,69 @@ public async void StartValidation()
     }
   }
 
+  /// <summary>
+  /// Helper class to handle the CLI options and arguments.
+  /// It is based on the CommandLine library.
+  /// </summary>
+  public class CLIParser {
+    ParserResult<CLIOptions> cliOptions;
+
+    /// <summary>
+    /// Data storage class to store the command line options and arguments.
+    /// </summary>
+    public class CLIOptions
+    {
+      [Option('s', "scope_dir", Required = false, HelpText = "Set the scope directory")]

```

Comment:
Let's go with `scopedir`, easier to type.

Reply:
Sure 👍